MobileWallet/Common/Theme.swift

```diff
@@ -326,9 +365,15 @@ struct Images: Loopable {
     let storeModal = UIImage(named: "store-modal")
     let unknownUser = UIImage(named: "unknownUser")
     let handWave = UIImage(named: "HandWave")
+    let attention = UIImage(named: "Attention")
+    let success = UIImage(named: "Success")
+    let tariIcon = UIImage(named: "TariIcon")
 
     //Amount
     let delete = UIImage(named: "numpad-delete")
```

Comment:
Didn't see these before but it looks like this area is missing a naming convention.